testdata/default/cheats/GetFoundryVersion.t.sol

```diff
@@ -10,17 +10,16 @@ contract GetFoundryVersionTest is DSTest {
     function testGetFoundryVersion() public view {
         string memory fullVersionString = vm.getFoundryVersion();
 
+        // Ensure the version string contains a "+" separator
         string[] memory versionComponents = vm.split(fullVersionString, "+");
-        require(versionComponents.length == 3, "Invalid version format");
+        require(versionComponents.length == 2, "Invalid version format");
 
+        // Validate semantic version (e.g., "0.3.0-stable")
         string memory semanticVersion = versionComponents[0];
         require(bytes(semanticVersion).length > 0, "Semantic version is empty");
 
+        // Validate commit hash (e.g., "3a9576857a")
         string memory commitHash = versionComponents[1];
         require(bytes(commitHash).length > 0, "Commit hash is empty");
-
-        uint256 buildUnixTimestamp = vm.parseUint(versionComponents[2]);
-        uint256 minimumAcceptableTimestamp = 202406111234;
-        require(buildUnixTimestamp >= minimumAcceptableTimestamp, "Build timestamp is too old");
```

Comment:
I think the point of this was that so it was possible to compare timestamps for certain features, I'm not sure how we can retain that if we remove build timestamp. Personally I don't mind keeping it

Reply:
alternatively one could check for git sha as optimisim doing here https://github.com/ethereum-optimism/optimism/pull/5987/files

Reply:
this would be a neat follow up https://github.com/foundry-rs/foundry/issues/5133

Reply:
Readded the timestamp, for ease of comparison I opted for using a unix timestamp for the semver compatible version rather than a custom format

The `--version` message uses a human readable version